service.dataplane.Dockerfile

```diff
@@ -32,9 +41,13 @@ RUN --mount=type=cache,target=/workspace/target/ \
     cargo xtask build-ebpf --release
 RUN --mount=type=cache,target=/workspace/target/ \
     --mount=type=cache,target=/root/.cargo/registry \
-    RUSTFLAGS=-Ctarget-feature=+crt-static cargo build --release --target=$(eval cat arch)-unknown-linux-musl
+    RUSTFLAGS=-Ctarget-feature=+crt-static cargo build \
+    --workspace \
+    --exclude ebpf \ 
```

Comment:
lets not exclude the ebpf crate for now. @astoycos  forgive me if i'm wrong but the bpfd/man integration is kinda broken atm?

Reply:
I need to re-verify, but I believe I _had_ to do this to get the build to work properly since ebpf is the only thing that's not explicitly managed by our workspace due to conflict that arose.

If we want to include ebpf, I may need to find a workaround.

Reply:
if the tests are passing, then i must be missing something, please ignore

Reply:
So this is fine since we're building the eBPF in the step before, however this container is the dataplane meaning we only need to build the loader here so excluding it is completely fine